### ISSUE TITLE
Advanced filtering for table data

### ISSUE THREAD

**Issue by DIMANRUS** [dbcodeio/public#40]
Hello. I would like to edit the SQL code after I open the table to select specific data and edit it, as it happens in Azure Data Studio and SSMS.
I can't do this through the filter, since the data that was received is filtered there, but I have a limit of 5000 rows and, for example, the record is not included in these rows.
I hope you understand what I wrote. Thank you.

**Comment by mikeburgh** [comment]
Yes, I follow what you mean.. 

The 5000 limit though, did you set that in settings ? default is 100,000 and you can change that by right clicking the table and choosing the open with limit option... Not a perfect fix if your table has 10 million rows.

**Comment by DIMANRUS** [comment]
Yes, I set this limit in the settings. But I have more than 100,000 rows in the tables, I don't want to cause problems for vs code with display))

**Comment by mikeburgh** [comment]
Yeah makes sense, let me think about how to make that workable from the UI side.. I want to try and keep all "SQL" inside a real vscode editor so you get all the editor benefits.. I could let you open it in a new tab, and then when you execute it the results will show in the panel, and you can then drag those into the window when #37  is done.. or maybe a keyboard modifier to execute to a new window rather than the panel (saves a few steps)

**Comment by mikeburgh** [comment]
This should be fixed now. Tables open with a limit of 100 by default, and sort/search re gets the data from the server so you can filter down what you need and then edit.
